packages/test/test-service-load/testConfig.json

```diff
@@ -5,7 +5,19 @@
             "progressIntervalMs": 5000,
             "numClients": 10,
             "totalSendCount": 1000,
-            "readWriteCycleMs": 10000
+            "readWriteCycleMs": 10000,
+            "optionOverrides":{
+                "tinylicious":{
+                    "configurations":{
+                        "Fluid.GarbageCollection.TestOverride.InactiveTimeoutMs": [5000]
```

Comment:
Are you going to set SessionExpiry override in a separate PR?

Reply:
yes, the idea is to validate current tests with inactive object timeout. Will add session timeout later.